store/store_test.go

```diff
@@ -732,33 +732,34 @@ func (t *remoteRepoTestSuite) TestUbuntuStoreFindAuthFailed(c *C) {
 }
 
 /* acquired via:
-curl -s --data-binary '{"name":["8nzc1x4iim2xj1g2ul64.chipaca"]}'  -H 'content-type: application/json' https://search.apps.ubuntu.com/api/v1/click-metadata
+$ curl -s --data-binary '{"snaps":[{"snap_id":"1e21e12ex4iim2xj1g2ul6f12f1","channel":"stable","revision":1,"confinement":"strict"}],"fields":["snap_id","package_name","revision","version","download_url"]}'  -H 'content-type: application/json' https://search.staging.apps.ubuntu.com/api/v1/metadata
 */
-const MockUpdatesJSON = `[
-    {
-        "status": "Published",
-        "name": "8nzc1x4iim2xj1g2ul64.chipaca",
-        "package_name": "8nzc1x4iim2xj1g2ul64",
-        "snap_id": "1e21e12ex4iim2xj1g2ul6f12f1",
-        "origin": "chipaca",
-        "changelog": "",
-        "icon_url": "https://myapps.developer.ubuntu.com/site_media/appmedia/2015/04/hello.svg_Dlrd3L4.png",
-        "title": "Returns for store credit only.",
-        "binary_filesize": 65375,
-        "anon_download_url": "https://public.apps.ubuntu.com/anon/download/chipaca/8nzc1x4iim2xj1g2ul64.chipaca/8nzc1x4iim2xj1g2ul64.chipaca_42_all.snap",
-        "allow_unauthenticated": true,
-        "revision": 3,
-        "version": "42",
-        "download_url": "https://public.apps.ubuntu.com/download/chipaca/8nzc1x4iim2xj1g2ul64.chipaca/8nzc1x4iim2xj1g2ul64.chipaca_42_all.snap",
-        "download_sha512": "5364253e4a988f4f5c04380086d542f410455b97d48cc6c69ca2a5877d8aef2a6b2b2f83ec4f688cae61ebc8a6bf2cdbd4dbd8f743f0522fc76540429b79df42"
+const MockUpdatesJSON = `{"_embedded":
+    {"clickindex:package": [
+        {"package_name": "hello-world.canonical",
```

Comment:
are we really getting back this? this seems very wrong (notice other examples have just hello-world), maybe the curl you are using needs to send release?